File: src/main/java/page/Page.java
package page;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.support.PageFactory;
import org.openqa.selenium.support.pagefactory.AjaxElementLocatorFactory;

public class Page {
    protected final WebDriver driver;

    public Page(final WebDriver driver) {
        this.driver = driver;
        PageFactory.initElements(new AjaxElementLocatorFactory(driver, 15), this);
    }
}

File: src/main/java/page/Page.kt
package page

import org.openqa.selenium.WebDriver
import org.openqa.selenium.support.PageFactory
import org.openqa.selenium.support.pagefactory.AjaxElementLocatorFactory

open class Page(driver: WebDriver) {
    init {
        PageFactory.initElements(AjaxElementLocatorFactory(driver, 15), this)
    }
}
